# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/notification/NotificationParent.h
# Commit: fca3921d09a5
# Full Hash: fca3921d09a59b709a6939f4a1e65f261367e0a6
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2024-11-12 09:32:50
# Regressor Bug: 1928702
# File Overlap Count: 1
# Description:
#   Bug 1928702 - Part 5: Implement RecvShow r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D227690
# ==============================================================================

diff -r 807f1d1ada66 -r fca3921d09a5 dom/notification/NotificationParent.h
--- a/dom/notification/NotificationParent.h	Mon Nov 11 20:34:12 2024 +0000
+++ b/dom/notification/NotificationParent.h	Mon Nov 11 20:34:13 2024 +0000
@@ -45,6 +45,8 @@
  private:
   ~NotificationParent() = default;
 
+  nsresult Show();
+
   void GetAlertName(nsAString& aRetval) {
     if (mAlertName.IsEmpty()) {
       MaybeInitAlertName();